tests/test_io.py

```diff
@@ -90,3 +96,39 @@ def test_clipboard_iterator():
         assert_type(read_clipboard(iterator=False, chunksize=1), TextFileReader),
         TextFileReader,
     )
+
+
+def test_hdf():
+    with ensure_clean() as path:
+        check(assert_type(DF.to_hdf(path, "df"), None), type(None))
+        check(assert_type(read_hdf(path), Union[DataFrame, Series, Index]), DataFrame)
+
+
+def test_hdfstore():
+    with ensure_clean() as path:
+        store = HDFStore(path, model="w")
```

Comment:
might be good using it as a context manager - tests also whether `__enter__` and `__exit__` are declared.

Reply:
Added a test